kernel/sched/litmus.c

```diff
@@ -338,6 +338,12 @@ static void set_next_task_litmus(struct rq *rq, struct task_struct *p)
 
 
 #ifdef CONFIG_SMP
+/* Basic no-op balance function
+ */
+static int 
+balance_litmus(struct rq *rq, struct task_struct *prev, struct rq_flags *rf){
```

Comment:
Nit: The rest of this file likes to use the style where the opening curly bracket is on the line after the function signature.